### DISCUSSION TITLE
Hi, I can't arm my quad speedybee f7 v2 fc. I have system message at osd "thr - ttle is not l - w nable to arm". Google search and search here have not results.

### DISCUSSION THREAD

**Discussion by Yarrikk** [iNavFlight/inav#7542] [Q&A] [answered]
**PLEASE MAKE SURE YOU READ AND UNDERSTAND THE SOCIAL MEDIA SUPPORT CHANNELS. QUESTIONS ABOUT FLASHING, CONFIGURING, PILOTING MAY BE CLOSED WITHOUT FURTHER INTERACTION.**

* [Telegram channel](https://t.me/INAVFlight)
* [Facebook group](https://www.facebook.com/groups/INAVOfficial)
* [RC Groups thread](https://www.rcgroups.com/forums/showthread.php?2495732-Cleanflight-iNav-%28navigation-rewrite%29-project)

**Please double-check that nobody reported the issue before by using search in this bug tracker.**

**PLEASE DELETE THE TEXT ABOVE AFTER READING AND UNDERSTANDING IT**

****

## Steps to Reproduce

1.
2.
3.
4.

## Additional context

---

- FC Board name and vendor:
- INAV version string:

**Comment by ryet9** [comment, marked as answer]
First make sure your throttle is low.... 😊 ..._OK_
A few things to check, in the **Receiver** tab make sure when you throttle up it's actually your throttle that moves, if not this could the "**Channel Map**" being wrong "**AETR**" & not "**TAER**" for example, or your sticks are not going LOW, perhaps change the weight on your Radio so it goes to 1000 & 2000 **low** & **high** (maybe set it as 998 as your low)
or better yet, use the "**rxrange**" command in the **CLI**, _Bardwell_ has a video that covers both of these & while he uses _**BetaFlight**_ to demonstrate it's the same for **_INAV_**
https://youtu.be/1azl1f8qJ84?t=525

_note: This has nothing to do with what type of Flight Controller you have, but the configuration of it._
